x/relationships/keeper/migrations.go

```diff
@@ -4,34 +4,23 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	v2 "github.com/desmos-labs/desmos/v4/x/relationships/legacy/v2"
-
-	profilesv4 "github.com/desmos-labs/desmos/v4/x/profiles/legacy/v4"
-
-	v1 "github.com/desmos-labs/desmos/v4/x/relationships/legacy/v1"
 )
 
 // DONTCOVER
 
 // Migrator is a struct for handling in-place store migrations.
 type Migrator struct {
 	keeper Keeper
-	pk     profilesv4.Keeper
 }
 
 // NewMigrator returns a new Migrator
-func NewMigrator(keeper Keeper, pk profilesv4.Keeper) Migrator {
```

Comment:
Remove profilesv4.Keeper since it occurs the circular dependency with `x/profiles` while setting app wiring